packages/financial-templates-lib/src/price-feed/BalancerPriceFeed.js

```diff
@@ -39,13 +40,20 @@ class BalancerPriceFeed extends PriceFeedInterface {
     }
     return this.priceHistory.get(block.timestamp);
   }
+
   getLastUpdateTime() {
     return this.lastUpdateTime;
   }
+
   getCurrentPrice() {
     // current price can be undefined, will throw for any other errors
     return this.priceHistory.currentPrice();
   }
+
```

Comment:
This method was added to the price feed (and others) to enable the `index.js` entry point to query the number of decimals within a price feed.